modules/requisition/src/main/java/org/openlmis/rnr/domain/RnrColumn.java

```diff
@@ -97,4 +96,8 @@ public boolean equals(Object o) {
     return (this.name.equals(rnrColumn.getName()));
   }
 
+  @Override
+  public int hashCode() {
+    return Objects.hash(this.name);
```

Comment:
This might be misleading.  Is the name truly unique among all R&Rs (and would it have to be), or unique within one R&R?